.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -824,12 +824,12 @@ $ TELEPORT_TEST_GCP_KMS_KEYRING=projects/<account>/locations/us-west3/keyRings/<
 
 ## Moderated session
 
-Using `tsh` join an SSH session as two moderators (two separate terminals, role requires one moderator).
- - [ ] `Ctrl+C` in the #1 terminal should disconnect the moderator.
- - [ ] `Ctrl+C` in the #2 terminal should disconnect the moderator and terminate the session as session has no moderator.
+Create two Teleport users, a moderator and a user. Configure Teleport roles to require that the moderator moderate the user's sessions. Use `TELEPORT_HOME` to `tsh login` as the user in one terminal, and the moderator in another.
 
-Using `tsh` join an SSH session as two moderators (two separate terminals, role requires one moderator).
-- [ ] `t` in any terminal should terminate the session for all participants.
+For each of the following cases, create a moderated session with the user using `tsh ssh` and join this session with the moderator using `tsh join --role moderator`:
+ - [ ] Ensure that `Ctrl+C` in the user terminal disconnects the moderator as the session has ended.
+ - [ ] Ensure that `Ctrl+C` in the moderator terminal disconnects the moderator and terminates the user's session as the session no longer has a moderator.
```

Comment:
This should only happen if the termination policy is `onLeave: terminate `. If it's `pause` , the session isn't terminated but it is paused and the user loses access to terminal until the moderator joins

Reply:
I'll add that as a note but not as another test case.